src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/ReducibleAnnotationBaseTest.java

```diff
@@ -100,15 +100,12 @@ public void testFinalizeAnnotationAgainstExpected(List<VariantContext> VCs, Vari
         }
 
         VariantAnnotatorEngine annotatorEngine = new VariantAnnotatorEngine(getAnnotationsToUse(), null, Collections.emptyList(), false, false);
-        final UnifiedArgumentCollection uac = new UnifiedArgumentCollection();
-        uac.genotypeArgs = new GenotypeCalculationArgumentCollection();
+        final StandardCallerArgumentCollection standardArgs = new StandardCallerArgumentCollection();
+        standardArgs.genotypeArgs = new GenotypeCalculationArgumentCollection();
```

Comment:
I don't think this needs to be set here anymore since it was just constructed with the same thing.

Reply:
fixed